### PR TITLE
more installing info in README.md

### PR DESCRIPTION
**Pull request by radames**
Give a bit of context on how to instal via NPM and via vanilla JS
responding to #69

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,36 @@ We use modern features to avoid polyfills and dependencies, so the libraries wil
...
+```
+
+### From CDN or Static hosting
+You can run our packages with vanilla JS, without any bundler, by using a CDN or static hosting. Using [ES modules](https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/), i.e. `<script type="module">`, you can import the libraries in your code:
+
+```html
+
+<script type="module">
+    import { HfInference } from 'https://cdn.jsdelivr.net/npm/@huggingface/inference@1/+esm';
+    import { createRepo, commit, deleteRepo, listFiles } from "https://cdn.jsdelivr.net/npm/@huggingface/hub@0/+esm";
```
Comment (coyotte508) [inline README.md:51]:
```suggestion
    import { HfInference } from 'https://cdn.jsdelivr.net/npm/@huggingface/inference/+esm';
    import { createRepo, commit, deleteRepo, listFiles } from "https://cdn.jsdelivr.net/npm/@huggingface/hub/+esm";
```

Maybe we need to add an import map for `@huggingface/hub` for `hash-wasm` when uploading files over 10MB in the commit call too

Comment (radames) [reply README.md:51]: thanks, I can add that example as well. Are you warning/throwing error depends on the size for the file? we could also add that info on the error message, wdyt?

Comment (coyotte508) [reply README.md:51]: We're not throwing any errors but we're using the `hash-wasm` dependency when dealing with big files in the browser

Comment (radames) [reply README.md:51]: ok, thanks, I did some tests with sourcemap and unfortunately its not working, the commit still works for small files, I guess we need a fix. Shall we merge this and update later?

Comment (coyotte508) [reply README.md:51]: Does adding something like this at the beginning work ?

```ts
<script type="importmap">
  {
    "imports": {
      "@huggingface/hub": "https://cdn.skypack.dev/@huggingface/hub",
      "hash-wasm": "https://cdn.skypack.dev/hash-wasm"
    }
  }
</script>
```

but even if it doesn't, I think it's ok to merge for now

Comment (radames) [reply README.md:51]: not really, :( I can create an issue on try slack , the bundler is not linking it correctly 
https://glitch.com/edit/#!/humble-confirmed-shadow

Comment (radames) [reply README.md:51]: sorry I think I forget to commit your suggestion above 🙈

**Pull request merged**